code/edgebridge/src/test/java/com/adobe/marketing/mobile/edge/bridge/EdgeBridgeExtensionTests.java

```diff
@@ -572,6 +573,118 @@ public void testHandleTrackEvent_trackAction_dispatchesEdgeRequestEvent() {
 		assertEquals(event.getUniqueIdentifier(), responseEvent.getParentID());
 	}
 
+	@Test
+	public void testHandleTrackEvent_nullAction_dispatchesEdgeRequestEvent() {
+		final Event event = new Event.Builder("Test Track Event", EventType.GENERIC_TRACK, EventSource.REQUEST_CONTENT)
+			.setEventData(
+				new HashMap<String, Object>() {
+					{
+						put("action", null);
```

Comment:
if state/action is invalid null/empty and there is no context data, I think we should not send this edge request

Reply:
Yes, thanks for the correction, no other data at all. 
If event data is empty altogether or, action and state and context data are null/empty and there is no additional data, then don't send an Edge request.